regen_perly.pl

```diff
@@ -90,7 +90,7 @@
 $version = $1;
 
 # creates $tmpc_file and $tmph_file
-my_system("$bison -d -o $tmpc_file $y_file");
+my_system("$bison -Wno-deprecated -d -o $tmpc_file $y_file");
```

Comment:
If we do deprecated stuff, perhaps we should document what exactly?

Reply:
It's the `%pure-parser` directive, which is deprecated in favor of '%define api.pure'
since Bison 2.3b (2008-05-27). We could declare that we only support 2.4 and newer (even RHEL 6 has 2.4.1).

This only affects people hacking on the grammar, not end-users.

Reply:
I've added a comment for now.